src/main/webapp/WEB-INF/pages/presentation.ftl

```diff
@@ -1,125 +1,220 @@
-                <a href="#demo3" data-toggle="collapse">
-                    <img src="images/development.png" class="img-circle services-icon" alt="Operational Expertise" width="400" height="400">
-                </a>
-                <div id="demo3" class="collapse">
-                    <p>Lorem ipsum dolor sit amet</p>
-                    <p>Fouded in 2016 </p>
-                    <p>read more ... </p>
-                </div>
-            </div>
-         </div>
-     </div>
-</section>  
-  
-<div id="portfolio" class="container-fluid text-center bg-grey">
-    <h2>Portfolio</h2><br>
-    <h4>What we have created</h4>
-    <br>
-  
-    <h2>What our customers say</h2>
-    <div id="myCarousel" class="carousel slide text-center" data-ride="carousel">
-        <!-- Indicators -->
-        <ol class="carousel-indicators">
-            <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
-            <li data-target="#myCarousel" data-slide-to="1"></li>
-            <li data-target="#myCarousel" data-slide-to="2"></li>
-        </ol>
+	<div id="save-sec">
+		<div class="row">
+			<div class= "col-md-10 col-md-offset-1 save-desc">
+				<div class="md-title">
+		    		Save BIG in your investments  
+			    </div> 
+			    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+			    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
+		    </div>
+	    </div>
+	    
+		<div class="save-mod"> 	
+		  <div class="row text-center">
+		    
+		    <div class="col-md-3 col-sm-5 col-centered">
+		      <div class="panel text-center">
+		        <div class="panel-heading price">
+		        	<h3>Save <span style="font-size:1.3em">&euro;50</span></h3>
+		        </div>
+		        <div class="panel-body">
+		          <p>Lorem ipsum</p>
+		          <p>Lorem ipsum</p>
+		          <p>Lorem ipsum</p>
+		          <p>Lorem ipsum</p>
+		        </div>
+		        <div class="panel-footer">
+		          <h4>Standard</h4>
+			          <a href="#user-login" data-toggle="modal"><button type="submit" class="stylish-btn stylish-sav-btn btn button-default">More details</button></a>
+		        </div>
+		      </div> 
+		    </div>  
+		   
+		   <div class="col-md-3 col-sm-5 col-centered">
+		       <div class="panel text-center">
+			        <div class="panel-heading price">
+			        	<h3>Save <span style="font-size:1.3em">&euro;25</span></h3>
+			        </div>
+			        <div class="panel-body">
+			            <p><strong>20</strong> Lorem</p>
+			            <p><strong>15</strong> Ipsum</p>
+			            <p><strong>5</strong> Dolor</p>
+			            <p><strong>2</strong> Sit</p>
+			        </div>
+			        <div class="panel-footer">
+			            <h4>Basic</h4>
+			            <a href="#user-login" data-toggle="modal"><button type="submit" class="stylish-btn stylish-sav-btn btn button-default">More details</button></a>
```

Comment:
nu e nevoie de button cu type submit, ajunge sa pui clasele de pe button pe a. La fel in celelalte 2 cazuri